src/police_vehicle_management/VehicleManagement.java

```diff
@@ -0,0 +1,149 @@
+		mostVehiclesProducer();
+
+		// Sort vehicles by vehicleNumber:
+		sortVehiclesByNumber();
+
+		// List quantity of management vehicles:
+		listVehiclesByType();
+	}
+
+	public static void addVehicle(Vehicle vehicle) {
+		System.out.println("**Adding vehicle");
+		if (vehicle.isValidVehicle()) {
+			vehicleList.add(vehicle);
+			System.out.println("Result: Add vehicle successfully");
+		} else {
+			System.out.println("Result: Add vehicle failed");
+		}
+	}
+
+	public static void findVehicleByNumber(String vehicleNumber) {
+		for (Vehicle vehicle : vehicleList) {
+			if (vehicle.getVehicleNumber() == vehicleNumber) {
+				System.out.println("**Vehicle by number - " + vehicle.getVehicleNumber());
+				vehicle.displayInfo();
+			}
+		}
+	}
+
+	public static void findVehicleByOnwerId(String id) {
+		for (Vehicle vehicle : vehicleList) {
+			if (vehicle.getVehicleOnwer().getId() == id) {
+				System.out.println("**Vehicle by onwer id - " + vehicle.getVehicleOnwer().getId());
+				vehicle.displayInfo();
+			}
+		}
+	}
+
+	public static void deleteVehiclesByProducer(String producer) {
+		int vehicleListLength = vehicleList.size();
+		System.out.println("Vehicles list before delete: " + vehicleListLength);
+		for (int i = 0; i < vehicleListLength; i++) {
+			if (vehicleList.get(i).getProducer() == producer) {
+				System.out.println("Delete vehicle: " + vehicleList.get(i).getVehicleNumber() + " successfully");
+				vehicleList.remove(i);
+				vehicleListLength -= 1;
+				i -= 1;
+			}
+		}
+		System.out.println("Vehicles list after delete: " + vehicleList.size());
+	}
+
+	public static void mostVehiclesProducer() {
+		Hashtable<String, Integer> hashMap = new Hashtable<String, Integer>();
+		hashMap.put("Honda", 0);
+		hashMap.put("Suzuki", 0);
+		hashMap.put("Toyota", 0);
+		hashMap.put("Yamaha", 0);
+		for (Vehicle vehicle : vehicleList) {
+			if (vehicle.getProducer() == "Honda") {
+				hashMap.put("Honda", hashMap.get("Honda") + 1);
+			} else if (vehicle.getProducer() == "Suzuki") {
+				hashMap.put("Suzuki", hashMap.get("Suzuki") + 1);
+			} else if (vehicle.getProducer() == "Toyota") {
+				hashMap.put("Toyota", hashMap.get("Toyota") + 1);
+			} else {
+				hashMap.put("Yamaha", hashMap.get("Yamaha") + 1);
+			}
+		}
+		String mostVehiclesProducer = null;
+		int maxValue = Integer.MIN_VALUE;
+		for (Map.Entry<String, Integer> entry : hashMap.entrySet()) {
+			if (entry.getValue() > maxValue) {
+				maxValue = entry.getValue();
+				mostVehiclesProducer = entry.getKey();
+			}
+		}
+		System.out.println("**The most vehicles producer is " + mostVehiclesProducer);
+	}
+
+	public static void sortVehiclesByNumber() {
+		Collections.sort(vehicleList, new Comparator<Vehicle>() {
+			@Override
+			public int compare(Vehicle o1, Vehicle o2) {
+				return o1.getVehicleNumber().compareTo(o2.getVehicleNumber());
+			}
+		});
+		System.out.println("**The vehicles after sorting by number:");
+		System.out.println(Collections.singletonList(vehicleList));
```

Comment:
```suggestion
		Collections.sort(vehicles);
		vehicles.forEach(vehicle -> vehicle.displayInfo());
```
thử cái này xem thanh niên